test/katex-spec.js

```diff
@@ -2680,15 +2680,54 @@ describe("A parser taking String objects", function() {
     });
 });
 
-describe("Unicode", function() {
-    it("should parse all lower case Greek letters", function() {
-        expect("αβγδεϵζηθϑικλμνξοπϖρϱςστυφϕχψω").toParse();
-    });
-
-    it("should parse 'ΓΔΘΞΠΣΦΨΩ'", function() {
-        expect("ΓΔΘΞΠΣΦΨΩ").toParse();
+describe("Unicode accents", function() {
+    it("should parse Latin-1 letters in math mode", function() {
+        // TODO(edemaine): Unsupported Latin-1 letters in math: ÅåÆÇÐØÞßæçðøþ
+        expect("ÀÁÂÃÄÈÉÊËÌÍÎÏÑÒÓÔÕÖÙÚÛÜÝàáâãäèéêëìíîïñòóôõöùúûüýÿ")
+        .toParseLike(
+            "\\grave A\\acute A\\hat A\\tilde A\\ddot A" +
+            "\\grave E\\acute E\\hat E\\ddot E" +
```

Comment:
I guess expanding to `\grave A` makes sense.  Can you add a MathML fixture test?  I'm curious to see if we recombine the accent with the letter when we render to MathML.

Reply:
They render exactly how accent commands do, which I learned is via `<mover accent="true">`.  I think this makes sense...  Test added.